e2e/nomostest/portforwarder/port_forwarder.go

```diff
@@ -110,32 +118,72 @@ func NewPortForwarder(
 func (pf *PortForwarder) LocalPort() (int, error) {
 	pf.mux.Lock()
 	defer pf.mux.Unlock()
-	if pf.localPort == 0 {
+	if pf.subprocess == nil {
 		err := pf.start()
-		return pf.localPort, err
+		return pf.subprocess.localPort, err
```

Comment:
looks like this could cause a null pointer exception if `start` fails to set the `subprocess`

Reply:
Indeed. Added error check.